internal/db/update/schema.go

```diff
@@ -306,28 +337,36 @@ SELECT COUNT(name) FROM sqlite_master WHERE type = 'table' AND name = 'schemas'
 // trigger the associated Update value. It's required that the minimum key in
 // the map is 1, and if key N is present then N-1 is present too, with N>1
 // (i.e. there are no missing versions).
-func NewFromMap(versionsToUpdates map[int]schema.Update) *SchemaUpdate {
+func NewFromMap(internalUpdates, externalUpdates map[int]schema.Update) *SchemaUpdate {
 	// Collect all version keys.
-	versions := []int{}
-	for version := range versionsToUpdates {
-		versions = append(versions, version)
-	}
 
-	// Sort the versions,
-	sort.Ints(versions)
+	schemaUpdate := &SchemaUpdate{}
+	for i, update := range []map[int]schema.Update{internalUpdates, externalUpdates} {
```

Comment:
As I've seen this pattern in a few places ` range []map[int]schema.Update{internalUpdates, externalUpdates}` is it worth making a data type that has a `ToOrderedSlice()` function that returns the map in the correct order?